hooks/tk-3dsmaxplus_actions.py

```diff
@@ -109,30 +109,35 @@ def execute_action(self, name, params, sg_data):
             "Parameters: %s. Shotgun Data: %s" % (name, params, sg_data)
         )
 
-        # resolve path
-        path = self.get_publish_path(sg_data)
-
-        # If this is an Alembic cache, then we can import that.
-        if path.lower().endswith(".abc"):
-            # Note that native Alembic support is only available in Max 2016+.
-            if app.engine._max_version_to_year(app.engine._get_max_version()) >= 2016:
-                self._import_alembic(path)
-            else:
-                app.log_warning(
-                    "Alembic imports are not available in Max 2015, skipping."
-                )
-        elif name == "merge":
-            self._merge(path, sg_data)
-        elif name == "xref_scene":
-            self._xref_scene(path, sg_data)
-        elif name == "texture_node":
-            self._create_texture_node(path, sg_data)
-        else:
-            try:
-                HookBaseClass.execute_action(self, name, params, sg_data)
-            except AttributeError as e:
-                # base class doesn't have the method, so ignore and continue
-                pass
+        if sg_data["type"] == sgtk.util.get_published_file_entity_type(self.sgtk):
```

Comment:
The hook used to assume that every entity passing through was a publish, which was not the case, so we're testing for being a publish to run our hook code or we call the base class.